File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        length = len(nums)
        ActualSum = length * (length + 1 )/2 
        mySum = sum(nums)  # More Pythonic way to sum the list
        
        return  int(ActualSum - mySum)

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int len = nums.length;
        int actualsum =  len * (len+1)/2 ;
        int oursum = 0 ;
        for(int num : nums){
            oursum += num ;
        }

        return actualsum - oursum ;
    }
}